### PR TITLE
test page preview

### PR DESCRIPTION
**Pull request by ocefpaf**
Note that, if this works, it won't work from a fork due to security reasons. That is why I'm sending this PR from `ioos/ioos_code_lab` instead. We need to think if this is worth it. The alternative is to use something like netfly but that will require another account, login, credentials, etc...

### CODE REVIEW

**Comment by ocefpaf** [comment]
The comment was not created but the temporary branch with the preview-changes was: https://github.com/ioos/ioos_code_lab/commit/a9b2ed16a89466d639aad8d9479aac07d8db516b

**Comment by ocefpaf** [comment]
@MathewBiddle I wonder if this is worth pursing due to the limitations on how the preview will show up. Maybe netfly is the way to go here. What do you think?

**Comment by MathewBiddle** [comment]
@ocefpaf could you provide a little more detail? Is this logged in an issue I could review?

It looks like your trying to add some feature called "preview" that...gives you a preview webpage of the intended changes?

A little more background would help me understand if this is something worth pursuing.

**Comment by ocefpaf** [comment]
> @ocefpaf could you provide a little more detail? Is this logged in an issue I could review?

Sorry. This is such an old request that I let fall through the cracks but I'm trying to catch up with those now. The original issue is probably in the old version of this repository. Indeed this is about getting a page preview and we never got to it due to the complexities in commit rights.

Back when we had the Monday meetings we did touch based on netfly and how that service could generate these previews for us. However, it required managing another account and I'm not sure they have a free tier, so we dropped it.

This GHA alternative could work but then we would need to send all PRs from the main repo instead of a fork, which is not recommended.

TL;DR I'm inclined to look at netfly again but if that doesn't work. Abandon this and move to the next long standing issue. (Adding the many notebooks in draft PRs!)

**Comment by MathewBiddle** [comment]
Do we need a preview? IMO the process using GHA builds quick enough that we can 'review' after major PRs are merged simply using the live website.

**Comment by ocefpaf** [comment]
> Do we need a preview? IMO the process using GHA builds quick enough that we can 'review' after major PRs are merged simply using the live website.

This was an old request and I guess that revisiting it in the era of notebooks diffs on GH, no. We don't need a preview. The rare case where one would be required is is we start changing the page theme, etc. Our PRs here are mostly Markdown files and notebooks, all diffable now.

**Comment by MathewBiddle** [comment]
Okay. I say we ditch this effort as no longer applicable.
